tck/running_the_tck.asciidoc

```diff
@@ -114,6 +114,9 @@ If you use Apache Maven, then the tests are run via the `maven-surefire-plugin`
                 <environmentVariables>
                     <MP_METRICS_TAGS>tier=integration</MP_METRICS_TAGS>
                 </environmentVariables>
+                <includes>
+                    <include>**/*Test.java</include>
```

Comment:
Why do we need this? 
This only opens up for the possibility that someone introduces a test class ending with `TestCase` or something and then it will be excluded and nobody will notice

Reply:
I'm not able to run the TCK by following the [running_the_tck.asciidoc](https://github.com/eclipse/microprofile-metrics/blob/master/tck/running_the_tck.asciidoc) until added the above `include` pattern.

See: http://maven.apache.org/surefire/maven-failsafe-plugin/integration-test-mojo.html#includes

 If `include` pattern not specified, the following default includes is assigned which doesn't match which TCK test classes name and no test executed in the goal maven-`failsafe-plugin:integration-test`:
````
<includes>
    <include>**/IT*.java</include>
    <include>**/*IT.java</include>
    <include>**/*ITCase.java</include>
</includes>
````

Reply:
Well but this is a configuration example for the `surefire` plugin and you're changing it due to something specific to the `failsafe` plugin. Such `<includes>` element will need to be used if you are using the `failsafe` plugin, this example is not about  that. We could potentially include an example for the `failsafe` plugin in case anyone wants to use it, but it would need to be separate.

Reply:
Thanks @jmartisk, for the clarification,